core/register_core_types.cpp

```diff
@@ -149,6 +149,9 @@ void register_core_types() {
 	ClassDB::register_virtual_class<InputEventGesture>();
 	ClassDB::register_class<InputEventMagnifyGesture>();
 	ClassDB::register_class<InputEventPanGesture>();
+	ClassDB::register_class<InputEventGesturePan>();
```

Comment:
What's the difference between `InputEventPanGesture` and `InputEventGesturePan`? That's confusing.
`InputEventMagnifyGesture` should likely be renamed to `InputEventGestureMagnify` too.

Reply:
InputEventMagnifyGesture and InputEventPanGesture are OSX specific classes that should go away